projects/Mallard/src/screens/settings/gdpr-consent-screen.tsx

```diff
@@ -275,14 +276,19 @@ const GdprConsent = ({
 }
 
 const GdprConsentScreen = ({ navigation }: NavigationInjectedProps) => (
```

Comment:
We should look to use `useNavigation` hook at the components where its used rather than passing down navigation through multiple components